secdiff.py

```diff
@@ -235,6 +238,7 @@ def run(scr, state):
             state.up()
         elif c == curses.KEY_RESIZE:
             curses.update_lines_cols()
+            renderer = Renderer()
```

Comment:
Why is it a better idea to recreate the Renderer every time instead of relying on `resize_windows()`? If this is simpler, we should just delete `resize_windows()`.

NB: I don't actually understand curses very well, I just did what was necessary to get this working last night.

Reply:
I don't understand anything either, but this line improved the behavior, so it is definitely better with it than without it.

Reply:
Alright I'll just merge this and deal with the consequences later.